src/GitLink/Pdb/SrcSrv.cs

```diff
@@ -29,12 +29,22 @@ public static byte[] Create(string rawUrl, string revision, IEnumerable<Tuple<st
                 using (var sw = new StreamWriter(ms))
                 {
                     var scheme = new Uri(rawUrl).Scheme;
-
+                
                     sw.WriteLine("SRCSRV: ini ------------------------------------------------");
                     sw.WriteLine("VERSION=2");
-                    sw.WriteLine("SRCSRV: variables ------------------------------------------");
-                    sw.WriteLine("SRCSRVVERCTRL={0}", scheme);
-                    sw.WriteLine("SRCSRVTRG={0}", CreateTarget(rawUrl, revision));
+                    sw.WriteLine("SRCSRV: variables ------------------------------------------");                    
+                    sw.WriteLine("RAWURL={0}", CreateTarget(rawUrl, revision));
+                    if(DebugWithPowershell)
+                    {
+                        sw.WriteLine("TRGFILE=%fnbksl%(%targ%%var2%)");
+                        sw.WriteLine("SRCSRVTRG=%TRGFILE%");
+                        sw.WriteLine("SRCSRVCMD=powershell -NoProfile -Command \"(New-Object system.net.WebClient).DownloadFile('%RAWURL%', '%TRGFILE%')\"");
```

Comment:
I think it's better to use right casing, even with powershell:

System.Net.WebClient